packages/corvid-e2e-tests/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "run-script-os",
     "test:windows": ":",
-    "test:default": "jest --maxWorkers=1 --detectOpenHandles"
```

Comment:
why the `runInBand` (which has performance penalties)? Is it a must for our e2e to work?
I see `detectOpenHandles` is also only meant to be used for [debugging](https://jestjs.io/docs/en/cli.html#--detectopenhandles) because it implies `runInBand`, why did we use it?

Reply:
i remember we had many flaky tests when running them in parallel.
i'll try to remove it and see if it's more stable now.